scipy/linalg/_special_matrices.py

```diff
@@ -331,6 +331,7 @@ def leslie(f, s):
     return a
 
 
+@xp_capabilities(jax_jit=False, skip_backends=[("dask.array", "dask/dask#1180")])
```

Comment:
>I think Dask can skip the test.

One test of a degenerate case fails due to a bug in Dask. I'd suggest that we skip the test rather than skipping all tests. I know this will mark Dask as supported, and I think that's OK. Dask is supported; Dask just happens to have a bug.